bootstrap.sh

```diff
@@ -101,13 +101,16 @@ fi
 echo "[+] Installing the disassembler"
 sudo python ${DIR}/tools/setup.py install
 
+PROCS=$(nproc)
+
 # Create makefiles
 echo "[+] Creating Makefiles"
 pushd ${BUILD_DIR}
 MCSEMA_DIR=$(realpath ${DIR})
 BUILD_DIR=$(realpath ${BUILD_DIR})
 LLVM_DIR=$(realpath ${LLVM_DIR})
 GEN_DIR=$(realpath ${GEN_DIR})
+INSTALL_DIR=${MCSEMA_DIR}
```

Comment:
Can you make this configurable somehow? Might need "proper" command line arg (e.g. `--prefix` for install dir) for bootstrap to handle install location and build target type?